.github/workflows/codeowners.yml

```
@@ -0,0 +1,19 @@
+name: Ping maintainers
+
+on:
+  pull_request:
+    types: [opened, synchronize, ready_for_review]
+  push:
+    types: [opened, synchronize, ready_for_review]
+
+jobs:
+  codenotify:
+    runs-on: ubuntu-latest
+    name: codenotify
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
```

So it uses an up to date `CODENOTIFY` rather than as it exists in the PR branch.

```suggestion
          ref: refs/pull/${{ github.event.pull_request.number }}/merge
```